.flake8.cython

```
@@ -1,4 +1,4 @@
 [flake8]
 filename = *.pyx, *.pxd, *.pxi
 exclude = .git, .eggs, *.py
-ignore = W503,W504,E225,E226,E227,E402,E999
+ignore = W503,W504,E225,E226,E227,E275,E402,E999
```

E275 is to silence errors at `except?` (`missing whitespace after keyword`).